server/container_create.go

```diff
@@ -62,6 +64,33 @@ func addOciBindMounts(sb *sandbox, containerConfig *pb.ContainerConfig, specgen
 	return nil
 }
 
+func addDevices(sb *sandbox, containerConfig *pb.ContainerConfig, specgen *generate.Generator) error {
+	sp := specgen.Spec()
+	for _, device := range containerConfig.GetDevices() {
+		d, err := devices.DeviceFromPath(device.HostPath, device.Permissions)
+		if err != nil {
+			return err
```

Comment:
I'm going back and forth about the need, so will throw this out for consideration.  Would there be any benefit to having a logrus.Debug statement in this if statement?  Do you think the err variable will have enough info in it to figure out what went south in case of an error?  I think there is enough info after looking at DeviceFromPath, but I'm not 100% sure.